Gemfile

```diff
@@ -158,7 +158,7 @@ group :test, :development do
 
   gem "rspec-rails"
 
-  gem "shoulda-matchers", require: false
+  gem "shoulda-matchers", require: false, github: "thoughtbot/shoulda-matchers"
```

Comment:
Why is this necessary? I see it's for `greater_than_or_equal_to` but have they not cut a new gem recently that includes that or something? We should probably have a `TODO` to go back and change this to the non-github version at some point.

Reply:
Yep no gem AFAICT, I make a todo topic